test/asgs/listASGsToStart.test.js

```diff
@@ -9,7 +9,7 @@ describe('listASGsToStart()', () => {
   it('returns list of asgs spun down by hammertime', () => {
     AWS.mock('AutoScaling', 'describeAutoScalingGroups', startOnePageResponse);
 
-    return listASGsToStart()
+    return listASGsToStart(10)
```

Comment:
This would be more obvious as a variable called "defaultTimeZone", thoughts?

Reply:
Indeed, will sort that out.